components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/models/Workflow.java

```diff
@@ -135,11 +137,20 @@ public void setAttributes(Map<String, String> attributes) {
         this.attributes = attributes;
     }
 
+    public String getCreatedBy() {
+        return createdBy;
+    }
+
+    public void setCreatedBy(String createdBy) {
+        this.createdBy = createdBy;
+    }
+
     @Override
     public String toString() {
         return "Workflow [workflowReference=" + workflowReference + ", workflowType=" + workflowType + ", status="
                 + status + ", createdTime=" + createdTime + ", updatedTime=" + updatedTime + ", workflowDescription="
-                + workflowDescription + ", callbackUrl=" + callbackURL + ", attributes=" + attributes + "]";
-    }    
+                + workflowDescription + ", externalWorkflowReference=" + externalWorkflowReference + ", callbackURL="
```

Comment:
Shall we use String builder for this as following.
Example : 
String a = b + c + d;
into
String a = new StringBuilder(b).append(c).append(d).toString();